helpers/lib/fonts.js

```diff
@@ -104,38 +96,41 @@ module.exports = function(format, themeSettings, handlebars, options) {
             collectedFonts[provider] = [];
         }
 
-        collectedFonts[provider].push(value);
+        collectedFonts[provider].push(themeSettings[setting]);
     });
 
     // Parse font strings based on provider
-    const parsedFonts = _.mapValues(collectedFonts, function(value, key) {
-        return fontProviders[key].parser(value);
-    });
+    const parsedFonts = Object.keys(collectedFonts).reduce((parsedFonts, font) => {
```

Comment:
reduce, while nice and easy to read, is going to be slower than a for loop: https://hackernoon.com/3-javascript-performance-mistakes-you-should-stop-doing-ebf84b9de951

Reply:
fixed.